### ISSUE TITLE
Install fails on Debian 10

### ISSUE THREAD

**Issue by pirolen** [proycon/LaMachine#190]
Hi, I wanted to install the development version on Debian 10. 
The first error was that the missing Ansible could not be installed, I solved this by manually installing it as explained [here](https://computingforgeeks.com/how-to-install-ansible-awx-on-debian-buster/).
The next error was
`'/usr/bin/aptitude safe-upgrade' failed: E: Sub-process /usr/bin/dpkg returned an error code (1)`.
I attach the log. Thanks in advance for hints!
[lamachine-lamadev.log](https://github.com/proycon/LaMachine/files/5724370/lamachine-lamadev.log)

**Comment by proycon** [comment]
This error seems not be caused by LaMachine as such, but by upgrade of an existing package you had no your debian system was in. LaMachine simply does an apt upgrade to update everything, so if that fails it will fail too. Your logs says:

```
Preparing to unpack .../gitlab-ee_13.6.3-ee.0_amd64.deb ...
Malformed configuration JSON file found at /opt/gitlab/embedded/nodes/badwveritservice2.srv.mwn.de.json.
This usually happens when your last run of `gitlab-ctl reconfigure` didn't complete successfully.
This file is used to check if any of the unsupported configurations are enabled,
and hence require a working reconfigure before upgrading.
Please run `sudo gitlab-ctl reconfigure` to fix it and try again.
dpkg: error processing archive /var/cache/apt/archives/gitlab-ee_13.6.3-ee.0_amd64.deb (--unpack):
new gitlab-ee package pre-installation script subprocess returned error exit status 1
Errors were encountered while processing:
/var/cache/apt/archives/gitlab-ee_13.6.3-ee.0_amd64.deb
```

Gitlab is not something LaMachine installs, I'd suggest following gitlab-ctl instructions it gives and try a manual ``apt update && apt upgrade`` to verify everything is in order, then you can update/install lamachine again.

**Comment by proycon** [comment]
I'm actually more interested in why the ansible installation failed, which you solved already, as that is something that LaMachine should manage correctly :)

**Comment by pirolen** [comment]
> This error seems not be caused by LaMachine as such, but by upgrade of an existing package you had no your debian system was in. 

> Gitlab is not something LaMachine installs, I'd suggest following gitlab-ctl instructions it gives and try a manual `apt update && apt upgrade` to verify everything is in order, then you can update/install lamachine again.

I see, thanks, I will see to it.

**Comment by pirolen** [comment]
> I'm actually more interested in why the ansible installation failed

Yes, solved it, since I already encountered this issue on Ubuntu 18, around October, while doing a LaMachine update. Btw it was an easy fix now on Debian, but on Ubuntu it was not and I was too shy back then to make an issue ;-) 
As far as I remember, there existed another version of Ansible already, by a different installer, so I had to manually deinstall it (similar to this [solution](https://stackoverflow.com/questions/60834692/how-to-completely-remove-ansible-2-8-3-on-ubuntu-18-04)) and manually install it again -- but I don't clearly remember.
(Was there perhaps some change in the LaMachine installation? Because when I installed LaMachine for the first time in the summer on the Ubuntu machine, this problem did not emerge. )
